test/integration_test/migration_test.go

```diff
@@ -1239,6 +1245,94 @@ func pvcResizeMigrationTest(t *testing.T) {
 	validateAndDestroyMigration(t, ctxs, preMigrationCtx, true, false, true, false, false)
 }
 
+func clonePvcDestTest(t *testing.T) {
+	var err error
+	var namespace string
+	var pvcs *v1.PersistentVolumeClaimList
+	// Reset config in case of error
+	defer func() {
+		err = setSourceKubeConfig()
+		require.NoError(t, err, "Error resetting remote config")
+	}()
+	err = setMockTime(nil)
+	require.NoError(t, err, "Error resetting mock time")
+	ctxs, preMigrationCtx := triggerMigration(
+		t,
+		"mysql-migration-schedule-interval",
+		"mysql-1-pvc",
+		[]string{"cassandra"},
+		[]string{"mysql-migration-schedule-interval"},
+		true,
+		false,
+		false,
+		false,
+		"",
+		nil,
+	)
+
+	validateMigration(t, "mysql-migration-schedule-interval", preMigrationCtx.GetID())
+
+	// Get pvc lists from source cluster
+	for _, spec := range ctxs[0].App.SpecList {
+		if obj, ok := spec.(*apps_api.StatefulSet); ok {
+			namespace = obj.GetNamespace()
+			pvcs, err = core.Instance().GetPersistentVolumeClaims(namespace, nil)
+			require.NoError(t, err, "error retriving pvc list from %s namespace", namespace)
+			break
+		}
+	}
+
+	sc_src, err := core.Instance().GetStorageClassForPVC(&pvcs.Items[0])
+	require.NoError(t, err, "failed to get storage class: %v", err)
+	sc_src_name := sc_src.ObjectMeta.Name
+	sc_src_rp := *sc_src.ReclaimPolicy
+	sc_src_bm := *sc_src.VolumeBindingMode
+	sc_src_pm := sc_src.Parameters
+	sc_src_pr := sc_src.Provisioner
+	err = setDestinationKubeConfig()
+	require.NoError(t, err, "failed to set kubeconfig to dest cluster: %v", err)
+	_, err = createStorageClassForPvc(sc_src_name, sc_src_rp, sc_src_bm, sc_src_pm, sc_src_pr)
```

Comment:
Use the returned storage class and use that every where.

Reply:
Thank you, addressed this comment.